<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Signup</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>20</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>20</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>331</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="firstname">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 2px solid black;
color:black;
padding-bottom: 7px</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>First Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="surname">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 2px solid black;
color:black;
padding-bottom: 7px</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Surname</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="username">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 2px solid black;
color:black;
padding-bottom: 7px</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="email">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 2px solid black;
color:black;
padding-bottom: 7px</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Email</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {border-radius: 14px;
color: white;
background-color: rgb(0, 0, 13);
font-weight: bold;
font-size: 15px;
font: 10pt &quot;Arial&quot;;
}
QPushButton:hover{
background-color: rgb(0, 0, 25); 
color: rgb(0, 153, 255);
font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>Next â†’</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(242, 242, 242);
border-radius: 1px;
}
QPushButton:hover{
text-decoration: underline;
}
</string>
    </property>
    <property name="text">
     <string>Already have an account? Login</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
